### PR TITLE
feat(Blog): add captions to images

### PR DESCRIPTION
**Pull request by hofmann-felix**
Preview URL: https://satellytescommain21751-featimagecaptions.gtsb.io/blog/gather-metaverse-bump-each-other-2021/

**What's included?**
- In https://github.com/satellytes/satellytes.com/issues/353, certain font styles are requested for the captions of images in a blog post.
- `gatsby-remark-images` displays the image captions to which we can thereby apply a certain style now. 

**Notes**
- Due to this change, the markdown of all blog posts must be adjusted. Until now I have done it only in one blog post. In case the changes are fine, I'll adjust all the other articles

**Preview**
Bildschirmfoto 2022-02-10 um 10 33 16

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -102,6 +102,7 @@ module.exports = {
               maxWidth: 1200,
               linkImagesToOriginal: true,
               withWebp: true,
+              showCaptions: true,
```
Comment (georgiee) [inline gatsby-config.js:105]: Great finding ✨

File: blog-posts/gather-metaverse-bump-each-other-2021.md [hunk 2]
```diff
@@ -38,16 +37,14 @@ This sounds a lot like [Second Life](https://en.wikipedia.org/wiki/Second_Life).
 ## How we use Gather
 We started using Gather exactly six weeks ago. Your first task is to create your own space. Luckily, they offer plenty of pre-made spaces to pick from. I created a virtual office space, where we could sit at desks or meet at the bar in a kitchen.
 
-![](images/gather/gather-templates.png)
-_Pick an existing gather space from plenty of templates_
+![Pick an existing gather space from plenty of templates](images/gather/gather-templates.png)
```
Comment (georgiee) [inline blog-posts/gather-metaverse-bump-each-other-2021.md:40]: Oh that's cool that this is presented as `figcaption` 🙌

File: src/components/legacy/markdown/custom-components.tsx [hunk 3]
```diff
@@ -105,6 +106,15 @@ const DetailsStyled = styled.details`
...
 `;
+
+const Figure = styled.figure`
+  margin: 0;
+`;
+
+const Figcaption = styled.figcaption`
+  ${TextStyles.textXS};
+  margin-top: 12px;
+`;
```
Comment (georgiee) [inline src/components/legacy/markdown/custom-components.tsx]: thanks for those new two components for the markdown👍

**Review by georgiee — APPROVED** [review]
Thank you, this is some simple but amazing change. Something we wanted for a long time 💪 Now we have the technical foundation plus a correct style. I love it.

I expect you have checked with the design team (maybe some note in the PR?) that's why I would approve this already. The single change is good enough for this PR and a following PR can handle the migration. Thank you!

File: blog-posts/gather-metaverse-bump-each-other-2021.md [hunk 4]
```diff
@@ -56,29 +53,26 @@ The meeting room behavior of Gather is fundamentally different to the rest of th
...
 
 ### Pair programming
 A similar effect happens when you sit in the virtual office and suddenly two folks group up on the other side of the table. Instantly you know, _"oh, they do some pair programming"_. When I asked my colleagues about their session, they said it's a nice way to meet at some actual (although virtual) location and then just _"bump into the other person"_ to start the session. 
 
 This feeling is the promise of the Metaverse. It brings back the informal human interactions and spontaneity. You don't have to create a video meeting upfront or send some specific invitation. Who would do that in real life? Instead, you just walk over to the desk of your colleague and start talking.  
 
-![](images/gather/gather-pair.gif)
-
-_Our colleagues Jesús & Felix having a pair programming session_
+![Our colleagues Jesús & Felix having a pair programming session](images/gather/gather-pair.gif)
```
Comment (feedm3) [inline blog-posts/gather-metaverse-bump-each-other-2021.md:70]: This gif doesn't have a caption. Is there something broken?

File: blog-posts/gather-metaverse-bump-each-other-2021.md [hunk 5]
```diff
@@ -56,29 +53,26 @@ The meeting room behavior of Gather is fundamentally different to the rest of th
...
 When you walk to someone, while approaching, the video and audio slowly fade in. That technique is called "spatial audio technology" and mimics the way we bump into other people in real life, where someone's voice is louder while you are near them, and softer as you walk away. This is wonderful because it feels so familiar and natural.
 
-<video preload autoplay loop width="100%">
-  <source src="images/gather/video.mp4" type="video/mp4" />
-  <source src="images/gather/video.mp4" type="video/mp4" />
-</video>
-
-_People bumping into each other (via [gather.town](http://www.gather.town))_
+<figure>
+    <video preload autoplay loop width="100%">
```
Comment (feedm3) [inline blog-posts/gather-metaverse-bump-each-other-2021.md]: The video doesn't play, it's just displayed as image.

**Review by feedm3 — CHANGES_REQUESTED** [review]
Looks awesome! I found 2 bugs. Let's see if they can be fixed :)

File: blog-posts/gather-metaverse-bump-each-other-2021.md (hunk 4 above)
Comment (hofmann-felix) [reply blog-posts/gather-metaverse-bump-each-other-2021.md:70]: @feedm3 thanks for the feedback 🙏 
Gifs and videos are not processed by `gatsby-remark-images`, but by `gatsby-remark-copy-linked-files`. That's why the captions are not displayed. As a workaround, you can include `<figcaption />` explicitly in the markdown. If you think of a better solution, please let me know.

File: blog-posts/gather-metaverse-bump-each-other-2021.md [hunk 6]
```diff
@@ -56,29 +53,27 @@ The meeting room behavior of Gather is fundamentally different to the rest of th
...
 When you walk to someone, while approaching, the video and audio slowly fade in. That technique is called "spatial audio technology" and mimics the way we bump into other people in real life, where someone's voice is louder while you are near them, and softer as you walk away. This is wonderful because it feels so familiar and natural.
 
-<video preload autoplay loop width="100%">
-  <source src="images/gather/video.mp4" type="video/mp4" />
-  <source src="images/gather/video.mp4" type="video/mp4" />
-</video>
-
-_People bumping into each other (via [gather.town](http://www.gather.town))_
+<figure>
+    <video preload autoplay loop muted width="100%">
```
Comment (feedm3) [inline blog-posts/gather-metaverse-bump-each-other-2021.md:57]: `muted` makes the video play in Chroma and Safari. Thanks for that!

**Review by feedm3 — APPROVED** [review]

**Pull request merged**